app/proguard.cfg

```diff
@@ -2,7 +2,6 @@
 -renamesourcefileattribute SourceFile
 -keepattributes *Annotation*
 -keepattributes SourceFile,LineNumberTable
--repackageclasses ''
```

Comment:
added this long ago to work around a support library bug that's probably fixed now.